src/Api/Controllers/CollectionsController.cs

```diff
@@ -44,7 +44,7 @@ public async Task<CollectionResponseModel> Get(Guid orgId, Guid id)
     }
 
     [HttpGet("{id}/details")]
-    public async Task<CollectionGroupDetailsResponseModel> GetDetails(Guid orgId, Guid id)
+    public async Task<CollectionAccessDetailsResponseModel> GetDetails(Guid orgId, Guid id)
     {
         if (!await ViewAtLeastOneCollectionAsync(orgId) && !await _currentContext.ManageUsers(orgId))
```

Comment:
I don't think it's something that's being handled specifically, might have too look into that. Theoretically the server should just not return any groups and then the dialog won't display them.

Reply:
As functions today, I believe the custom Manage Users role allows the member to edit other member's roles, collection access and group access (except when editing a owner or admin, this is restricted). But the custom manager users role does not grant access to the manage collection or groups pages. 

I've logged a lot of the nuances in the [custom permission structure on confluence](https://bitwarden.atlassian.net/wiki/spaces/Des/pages/176292811/Organization+Permission+Matrix).

Reply:
I've added special handling of groups into the frontend. It will no longer fetch groups and it will display other i18n strings that don't mention groups if their organization doesn't use groups.